src/inspector_agent.cc

```diff
@@ -215,7 +215,7 @@ class JsBindingsSessionDelegate : public InspectorSessionDelegate {
     Local<Value> argument = v8string.ToLocalChecked().As<Value>();
     Local<Function> callback = callback_.Get(isolate);
     Local<Object> receiver = receiver_.Get(isolate);
-    callback->Call(env_->context(), receiver, 1, &argument);
+    static_cast<void>(callback->Call(env_->context(), receiver, 1, &argument));
```

Comment:
Wouldn't 
```cc
#pragma GCC unused-result push
#pragma GCC unused-result ignored
callback->Call(env_->context(), receiver, 1, &argument)
#pragma GCC unused-result pop
```
be better, so it could be traced in the future

Or even just a comment stating the the `// static_cast<void> is to skip 'unused-result' warning`?